toonz/sources/toonz/main.cpp

```diff
@@ -161,6 +161,7 @@ DV_IMPORT_API void initColorFx();
 static void initToonzEnv() {
   StudioPalette::enable(true);
 
+  TEnv::setWorkingDirectory();
```

Comment:
How about calling `EnvGlobals::setWorkingDirectory()` in the constructor of `EnvGlobals`, instead of calling it externally here?
By doing this, I think we can share the portable-mode with other command-line binaries such as `tcomposer`.

Reply:
Good idea.

Reply:
I'm making the changes now.  What are your thoughts on changing the portablestuff folder to just stuff?  I don't think any platforms have the stuff folder in the app folder by default.  portablestuff feels a little long, but I can go either way.

Reply:
@turtletooth Thank you for the modifications. Well, I prefer "portablestuff" as it is more self-describing.

Reply:
Cool.  I'm good with that.

Reply:
If you are feeling daring, you could take a look at the sound pr.  I think I fixed it.  Development on a mac isn't nearly as fun as Visual Studio.  Are you using Qt Creator or XCode on Mac?

Reply:
I'm checking the sound PR now. Thanks for fixing that too.
I'm using CLion for OSX development, I totally agree that it is not comfortable as working on MSVC.